htdocs/class/theme.php

```diff
@@ -282,6 +282,13 @@ public function xoInit($options = array())
         if (strpos($xoops_page, 'modules') !== false) {
             $xoops_page = str_replace('modules/', '', $xoops_page);
         }
+		$tempScriptname = str_replace('\\', '/',  $_SERVER['SCRIPT_NAME']);
+		$tempRequesturi = str_replace('\\', '/',  $_SERVER['REQUEST_URI']);
```

Comment:
The 'REQUEST_URI'  can contain user input. We should use Request::getString() to retrieve the value..